src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -869,6 +869,10 @@
         - name: --enable-disk-driver
           type: bool
           short-summary: Enable AzureDisk CSI Driver.
+        - name: --ip-families
+          type: string
+          short-summary: A comma separated list of IP versions to use for cluster networking.
```

Comment:
Please fix the short summary since it is space-separated now.

Reply:
That's the only comment I have for this PR, and everything else looks good to me.

Reply:
so does the user need to input `--ip-families ipv4 ipv6` and it won't support `--ip-families ipv4,ipv6` anymore? isn't that somewhat of a breaking change?

Reply:
bump @yanzhudd or @zhoxing-ms

Reply:
maybe @FumingZhang could answer?

Reply:
That sounds like a breaking behavior change, please confirm with PM if this is acceptable.

Reply:
we don't want the change if nargs means it has to be space separated. @yanzhudd please confirm

Reply:
Hi @tyler-lloyd,
sorry for the delay reply. I thought it was a newly added param and not a breaking change, hence the suggestion. If it is a breaking change, just keep it and do not change it to space separated, thanks!

Reply:
sounds good. reverted the nargs change and keeping it comma separated